src/layouts/Layout.jsx

```diff
@@ -7,8 +7,8 @@ import Box from '@mui/material/Box';
 import useMediaQuery from '@mui/material/useMediaQuery';
 import { useTheme } from '@mui/system';
 // Component Imports
+import { InactivityMessage, NotificationContainer } from '@components/Notification';
```

Comment:
if this change was solely for the `@components`, should we do for `NavBar` as well?

Reply:
Sure. Think we could also create a new `index.js` file for `<Footer>` as well.